src/components/LanguageMenu/LanguageMenu.tsx

```diff
@@ -19,6 +18,7 @@ const languageMapper = (
 ): JSX.Element[] => {
```

Comment:
It might make sense to go through the code base and make sure such functions have a very similar name i.e. `*Mapper` to improve readability.